patches/services-device-geolocation-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/services/device/geolocation/BUILD.gn b/services/device/geolocation/BUILD.gn
+index 89da086564f98..290c720859a5e 100644
+--- a/services/device/geolocation/BUILD.gn
++++ b/services/device/geolocation/BUILD.gn
+@@ -2,6 +2,7 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//brave/services/device/geolocation/sources.gni")
```

Comment:
these patches should have been done in a single line, but I don't think they are needed at all. See https://github.com/brave/brave-core/pull/18063/files#r1190502333